### PR TITLE
-User now has maxRange, Deleted Dead code. On initialization maxRange…

### PR DESCRIPTION
**Pull request by JoMuelle**
-Very small PR, backend for choosing maxRange for matches
-on creation user maxRange is 10km
merge with 154

### CODE REVIEW

File: src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java [hunk 1]
```diff
@@ -264,6 +265,9 @@ public void updatePreferences(User user){
       if(!Objects.isNull(user.getGenderPreferences()) && !user.getGenderPreferences().isEmpty()){
           userToUpdatePreferences.setGenderPreferences(user.getGenderPreferences());
       }
+      if(user.getMaxRange()>1 && user.getMaxRange() < 20010){
+          userToUpdatePreferences.setMaxRange(user.getMaxRange());
+      }
```
Comment (daveking7) [inline src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java:270]: What happens if I pass maxRange = 1?

**Review by daveking7 — COMMENTED** [review]
Please wait until @anhuwi has merged his PR.

File: src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java (hunk 1 above)
Comment (JoMuelle) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java:270]: fixed with >=

**Comment by JoMuelle** [comment]
Ill wait with merging until andy has merged

**Pull request merged**